data/json/items/appliances/appliances_generators.json

```diff
@@ -2,50 +2,18 @@
   {
     "type": "GENERIC",
     "category": "other",
-    "id": "active_backup_generator",
-    "name": { "str_sp": "makeshift generator" },
-    "looks_like": "f_active_backup_generator",
-    "description": "A ubiquitous piece of compact machinery meant for running anything from the bathroom lights to electric stoves and other necessities when grid electricity is unavailable.  It just needs to be plugged in, fueled up, and then turned on.",
-    "weight": "200 kg",
-    "to_hit": { "grip": "bad", "length": "long", "surface": "any", "balance": "clumsy" },
-    "material": [ "steel" ],
-    "volume": "200 L",
-    "longest_side": "80 cm",
-    "price": "350 USD",
-    "price_postapoc": "25 USD",
-    "//1": "Steel jerrycan (jerrycan_big) values. Must have this pocket, otherwise will not have any fuel capacity as an appliance.",
-    "pocket_data": [
-      {
-        "pocket_type": "CONTAINER",
-        "rigid": true,
-        "watertight": true,
-        "max_contains_volume": "20 L",
-        "max_item_volume": "32 ml",
-        "max_contains_weight": "25 kg",
-        "item_restriction": [ "diesel", "biodiesel", "jp8", "lamp_oil", "motor_oil" ]
-      }
-    ],
-    "symbol": "O",
-    "color": "green_white",
-    "//2": "Prevent loading items into the generator while it's not in appliance form.",
-    "flags": [ "NO_RELOAD" ],
-    "melee_damage": { "bash": 5 }
-  },
-  {
-    "type": "GENERIC",
     "id": "portable_generator",
     "//": "https://www.amazon.com/dp/B0D6PM5XN1",
-    "copy-from": "active_backup_generator",
     "name": { "str_sp": "portable gasoline generator" },
     "looks_like": "f_active_backup_generator",
     "description": "Small, portable generator, one that people buy for a camp.  It just needs to be plugged in, fueled up, and then turned on.",
     "weight": "21817 g",
-    "to_hit": { "grip": "none", "length": "short", "surface": "any", "balance": "clumsy" },
+    "to_hit": { "grip": "solid", "length": "short", "surface": "any", "balance": "clumsy" },
```

Comment:
```suggestion
    "to_hit": { "grip": "none", "length": "short", "surface": "any", "balance": "clumsy" },
```

You're NOT getting a solid grip on a 21kg generator

Reply:
you do?
![71gRR76+2QL _AC_SX679_](https://github.com/user-attachments/assets/8c251bec-f6b3-4e39-919f-3bab272afe9b)
It is heavy, but you have a dedicated hand to hold it, it's grip is fine

Reply:
shouldn't `balance` represent the weight of the item? sure, you won't swing with it very well, but only because it's heavy, not because you don't have a good place to hold it. Make it empty, make it weight 100 g, and suddenly grip would be solid? i don't think that's how this field should work

Reply:
> shouldn't `balance` represent the weight of the item? sure, you won't swing with it very well, but only because it's heavy, not because you don't have a good place to hold it. Make it empty, make it weight 100 g, and suddenly grip would be solid? i don't think that's how this field should work

I agree with you, which is why I said you should be able to keep the grip solid if you make it two handed. But with one hand, I don't think it'd be solid.

Reply:
You are wrong, but i don't want to spend my time arguing over it